intel_mraa_write_gpio_block.py

```diff
@@ -0,0 +1,33 @@
+import mraa
+from nio.common.signal.base import Signal
+from nio.common.discovery import Discoverable, DiscoverableType
+from nio.metadata.properties import ExpressionProperty
+from .intel_mraa_gpio_base import IntelMraaGpioBase
+
+
+@Discoverable(DiscoverableType.block)
+class IntelMraaWriteGpio(IntelMraaGpioBase):
+
+    """ Use Intel's libmraa to interface with the IO on various platforms """
+
+    value = ExpressionProperty(title='Pin Value', default='{{ $value }}')
+
+    def _pin_mode(self):
+        return mraa.DIR_OUT
+
+    def _process_signal(self, signal):
+        write_value = self._get_write_value(signal)
+        signal.write_status = self._gpio_pin.write(write_value)
+        return signal
+
+    def _get_write_value(self, signal):
```

Comment:
The point of this method should be documented a little better I think. Its goal sort of seems to be to return a boolean but then if it can't it just returns whatever the value was.

The specific example I'm curious about is if `self.value(signal)` evaluated to an empty string. It would fail the integer cast, so an empty string would be returned. Is that going to break the `gpio.write` call? I will create an issue for this, just putting a note here for reference